nautobot/dcim/api/serializers.py

```diff
@@ -213,7 +213,6 @@ class ConnectedEndpointSerializer(PathEndpointModelSerializerMixin):
 
 class LocationTypeSerializer(NautobotModelSerializer, TreeModelSerializerMixin):
     url = serializers.HyperlinkedIdentityField(view_name="dcim-api:locationtype-detail")
-    parent = NestedLocationTypeSerializer(required=False, allow_null=True, default=None)
```

Comment:
My concern here is leaving `parent` to drf here which is a self-referential `Foreign Key` field, might cause some inconsistent data format/layout here. for example the `content_types` field here for the `LocationType` itself and its parent. We could just not override the `content_types` field here tho
![Screen Shot 2023-03-27 at 11 30 46 PM](https://user-images.githubusercontent.com/46973263/228120675-0ca6734d-8610-4632-8bb1-b5299190338d.png)

Reply:
Hmm, this is odd. Is DRF not able to find the `LocationTypeSerializer` to use for the `parent`?